setup.py

```diff
@@ -34,7 +34,7 @@ def read_file(filename, encoding='utf8'):
 # specified here to avoid masking the more specific request requirements in
 # acme. See https://github.com/pypa/pip/issues/988 for more info.
 install_requires = [
-    'acme>=0.29.0',
+    'acme[dev]',
```

Comment:
In fact in `setup.py`, you still need to set a "real" dependency, with version, and out of dev scope. Here, to rely on the development version of `acme`, put `acme>=0.37.0.dev0`. It will be updated automatically during the release.